### PR TITLE
[Doc] Documentation about deploying and configuring pulsar functions-worker

### PR DESCRIPTION
**Pull request by sijie**

### CODE REVIEW

File: site2/docs/functions-worker.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+sidebar_label: Functions Worker
+---
+
+Pulsar `functions-worker` is the component to run Pulsar Functions in cluster mode.
+This document describes the options, steps and configurations to set up Pulsar
+`functions-worker`.
+
+Before continuing the documentation, please be noted with a few things:
+
+1. The `--- Service Uris ---` lines in the following diagrams represents that Pulsar
```
Comment (Jennifer88huang) [inline site2/docs/functions-worker.md]: Service URLs?

Comment (sijie) [reply site2/docs/functions-worker.md]: fixed

**Comment by sijie** [comment]
@jerrypeng @srkukarni @Jennifer88huang  please help review it

**Comment by Jennifer88huang** [comment]
Sure, got it.

File: site2/docs/functions-worker.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+This section illustrates how to run `functions-worker` as a separated process in separated machines.
+
+![assets/functions-worker-separated.png](assets/functions-worker-separated.png)
+
+> NOTE: In this mode, you need to make sure `functionsWorkerEnabled` is set to `false` so that your brokers
+don't start `functions-worker` by mistake.
+
+#### Configure Functions-Worker for running separately
+
+Following are the instructions for configuring function-worker.
```
Comment (Jennifer88huang) [inline site2/docs/functions-worker.md]: Could we refine it as "To run function-worker separately, you have to configure the following parameters. "

File: site2/docs/functions-worker.md [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+
+##### Worker settings
```
Comment (Jennifer88huang) [inline site2/docs/functions-worker.md]: Could we just use "Worker parameters"? And same for the following titles.

**Comment by sijie** [comment]
@Jennifer88huang 

> Could we re-organize the structure as follows:

Your proposed structure looks good.

**Comment by merlimat** [comment]
run java8 tests

**Pull request merged**